cmd/manager/main.go

```diff
@@ -64,7 +61,8 @@ func main() {
 	printVersion()
 	log.Info(fmt.Sprintf("Running in development mode: %v", isDevelopmentEnv))
 
-	namespace, err := k8sutil.GetWatchNamespace()
+	// TODO: Remove once migrated to new scafold
```

Comment:
What does this mean? What is "new scaffold"? Please be specific in the comment, provide links if needed.

Reply:
Done.